checkov/kubernetes/checks/resource/base_registry.py

```diff
@@ -6,7 +6,7 @@ def __init__(self, report_type: str):
         super().__init__(report_type)
 
     def extract_entity_details(self, entity):
-        kind = entity["kind"]
+        kind = entity.get("kind")
```

Comment:
is it ok, if it is `None`? not that it results in errors in other areas downstream 😄

Reply:
The only place this method is called from is [here](https://github.com/bridgecrewio/checkov/blob/master/checkov/kubernetes/checks/resource/base_registry.py#L14), and it means that it would not be able to get checks by entity type and that the results for this unknown entity would be empty

Reply:
ok, gotcha. This is more of a type hint concern, but leave it as it is, I will see how it will be, when I add type hints to `kubernetes`, because `get_checks()` actually expects a `str`, which it isn't, but it also doesn't fail, which makes sense from the code.
```suggestion
        kind = entity.get("kind") or ""
```
maybe this would be better for downstream handling